src/PKSim.Presentation/Presenters/Charts/ChartPresenter.cs

```diff
@@ -51,7 +53,8 @@ public abstract class ChartPresenter<TChart, TView, TPresenter> : OSPSuite.Prese
          IIndividualPKAnalysisPresenter pkAnalysisPresenter,
          IChartTask chartTask, 
          IObservedDataTask observedDataTask, 
-         IChartUpdater chartUpdater, bool useSimulationNameToCreateCurveName)
+         IChartUpdater chartUpdater, bool useSimulationNameToCreateCurveName,
```

Comment:
align this parameter to the next line (old code but still, why not)

Reply:
not sure how you mean... should "IUserSettings userSettings)" go to the line above?

Reply:
Lol no
The useSimulationNameToCreateCurve

But that's not a big deal.